.github/workflows/test_upb.yml

```diff
@@ -171,10 +180,10 @@ jobs:
           # a single wheel. As a result we can just test the oldest and newest
           # supported Python versions and assume this gives us sufficient test
           # coverage.
-          - { os: ubuntu-latest, python-version: "3.8", architecture: x64, type: 'binary' }
-          - { os: macos-12, python-version: "3.8", architecture: x64, type: 'binary' }
-          - { os: ubuntu-latest, python-version: "3.12", architecture: x64, type: 'binary' }
-          - { os: macos-13, python-version: "3.12", architecture: x64, type: 'binary' }
+          - { os: ubuntu-latest, python-version: "3.8", architecture: x64, type: 'binary', presubmit: true }
```

Comment:
Can we flip the polarity of these?  I think it's safer to default to "always run" and have to explicitly opt out "continuous only" tests